bin/main.ts

```diff
@@ -4,10 +4,14 @@ import * as cdk from '@aws-cdk/core';
 // Sample construct
 import BlueprintConstruct from '../examples/blueprint-construct'
 
+// Team directories for network policy.
+const burnhamPolicyDir = './examples/teams/team-burnham/'
+const rikerPolicyDir = './examples/teams/team-riker/'
+const teamPolicyDirList = [burnhamPolicyDir,rikerPolicyDir]
```

Comment:
Lets move these into the `BlueprintConstruct` implementation. that way we don't have to add an additional property.

Reply:
Move to the `BlueprintConstruct`